CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v5.4.0 - 2025-05-14
+
+### Highlights
+
+Kotlin bindings only: transactions are now [`NonCancellable`](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-non-cancellable/),
+as [required by Uniffi](https://mozilla.github.io/uniffi-rs/latest/futures.html#cancelling-async-code). This prevents a category of bug where Kotlin thinks a transaction
+has been cancelled, while Rust thinks it is still running.
+
+### Bug Fixes
+
+- prevent cancellations during transactions (88a2e39)
+- prevent cancellations during transactions (fa60502)
```

Comment:
:shrug:

Reply:
Yeah, I know, I didn't want to nitpick his commit messages.